app/assets/javascripts/surveys.js

```diff
@@ -194,15 +202,29 @@
       userSurveys.trackEvent(survey.identifier, 'banner_shown', 'Banner has been shown')
     },
 
+    initialiseModuleOnElement: function (element, Module) {
+      if (Module) {
+        var module = new Module(element)
+        module.init()
+        return module
```

Comment:
Why is `module` returned here? I can't see it being used anywhere.

Reply:
Previously I thought I needed it as I thought I needed to call a function on one of the modules after initialising, but this wasn't the case. I left in `return module` to make the function generic just so it can be used by others in the future, but I don't mind if you want it removed :+1:

Reply:
I think let's take it out unless we need it right now - it can always be added in again if required later.